### PR TITLE
fix(PyGeosExporter): export absl::string_view

### PR DESCRIPTION
**Pull request by francoisbonneau**

### CODE REVIEW

File: bindings/python/tests/model/test-py-geos.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+if __name__ == '__main__':
+    geosciences_io.GeosciencesIOModelLibrary.initialize()
+    test_dir = os.path.dirname(__file__)
+    data_dir = os.path.abspath(os.path.join(
+        test_dir, "../../../../tests/data"))
+
+    model = geosciences.load_structural_model(
+        os.path.join(data_dir, "test.lso"))
+    exporter = geosciences_io.BRepGeosExporter(model,"brep_to_geos")
+    #exporter.run()
```
Comment (BotellaA) [inline bindings/python/tests/model/test-py-geos.py:43]: Why the run() are commented?

**Review by BotellaA — APPROVED** [review]

File: bindings/python/tests/model/test-py-geos.py (hunk 1 above)
Comment (francoisbonneau) [reply bindings/python/tests/model/test-py-geos.py:43]: because it does not work on stratigraphic model. I need to further investigate.

**Pull request merged**